client/lib/domains/email-forwarding/get-domains-with-email-forwards.js

```diff
@@ -0,0 +1,14 @@
+import { hasEmailForwards } from 'calypso/lib/domains/email-forwarding';
```

Comment:
This was previously a selector located at `client/state/selectors/get-email-forwards.js`. The way that selector used to work appears to have been the reason for the stray `QueryEmailForwards` components we saw.

With this new approach, we rely solely on domain objects (not also on the `emailForwards` state tree) to produce a return value. This means that we can remove the Redux reducer entirely.

Reply:
This is awesome, and a very simple adaptation that doesn't require the state tree and hence can be use anywhere.
However, I think it'll be nice and consistent to keep the name of the file and function algned i.e.

It's predictable that the file `get-domains-with-forwards.(ts|js)` should define a symbol `getDomainsWithForwards`